### PR TITLE
Login form style

### PR DESCRIPTION
**Pull request by mariano-zeppa**

### CODE REVIEW

File: client/features/auth/Login.jsx [hunk 1]
```diff
@@ -28,22 +28,25 @@ function Login ({ history }) {
...
-        </label>
-        <button type="submit">Log in</button>
-      </form>
-      <Link to="/register"><p>No Account? Register here</p></Link>
+    <div className="h-full">
+      <div className="h-full shadow-2xl">
+        <div className="flex flex-col w-screen h-full px-10 py-32 text-xl bg-indigo-500">
+          <form className="items-center w-full mx-auto my-auto" onSubmit={handleLogin}>
+            <div className="flex items-center bg-indigo-300 border-b border-indigo-500 rounded-t-lg">
+              <label htmlFor="name" className="w-20 ml-2 mr-8 text-pink-100">Email</label>
```
Comment (sirchras) [inline client/features/auth/Login.jsx]: label htmlfor needs to match input id

**Review by sirchras — COMMENTED** [review]
we need to get rid of the .DS_Store files, and maybe look at the Login form fields

**Review by AndKenneth — APPROVED** [review]
Looks good, I just tidied up a few random diffs and got rid of a few files.

**Comment by sirchras** [comment]
just made sure htmlfors matched ids

**Review by sirchras — APPROVED** [review]

File: client/features/auth/Login.jsx (hunk 1 above)
Comment (sirchras) [reply client/features/auth/Login.jsx]: fixed these

**Pull request merged**